packages/compass-components/package.json

```diff
@@ -34,44 +34,44 @@
     "@dnd-kit/core": "^6.0.7",
     "@dnd-kit/sortable": "^7.0.2",
     "@dnd-kit/utilities": "^3.2.1",
-    "@leafygreen-ui/badge": "^8.1.3",
-    "@leafygreen-ui/banner": "^8.0.1",
-    "@leafygreen-ui/button": "^21.3.0",
-    "@leafygreen-ui/card": "^11.0.0",
-    "@leafygreen-ui/checkbox": "^13.1.2",
-    "@leafygreen-ui/code": "^15.0.0",
-    "@leafygreen-ui/combobox": "^10.0.0",
-    "@leafygreen-ui/confirmation-modal": "^5.2.1",
-    "@leafygreen-ui/emotion": "^4.0.8",
-    "@leafygreen-ui/guide-cue": "^6.0.0",
-    "@leafygreen-ui/hooks": "^8.3.0",
-    "@leafygreen-ui/icon": "^12.8.0",
-    "@leafygreen-ui/icon-button": "^15.0.23",
-    "@leafygreen-ui/info-sprinkle": "^3.0.0",
-    "@leafygreen-ui/leafygreen-provider": "^3.2.0",
-    "@leafygreen-ui/logo": "^9.2.0",
-    "@leafygreen-ui/marketing-modal": "^4.2.3",
-    "@leafygreen-ui/menu": "^27.0.0",
-    "@leafygreen-ui/modal": "^16.1.0",
-    "@leafygreen-ui/palette": "^4.1.1",
-    "@leafygreen-ui/pipeline": "^6.0.0",
-    "@leafygreen-ui/polymorphic": "^2.0.2",
-    "@leafygreen-ui/popover": "^12.0.0",
-    "@leafygreen-ui/portal": "^5.1.1",
-    "@leafygreen-ui/radio-box-group": "^13.0.2",
-    "@leafygreen-ui/radio-group": "^11.0.3",
-    "@leafygreen-ui/search-input": "^4.0.0",
-    "@leafygreen-ui/segmented-control": "^9.0.0",
-    "@leafygreen-ui/select": "^13.0.0",
-    "@leafygreen-ui/table": "^12.7.0",
-    "@leafygreen-ui/tabs": "^13.1.1",
-    "@leafygreen-ui/text-area": "^9.1.2",
-    "@leafygreen-ui/text-input": "^13.1.2",
-    "@leafygreen-ui/toast": "^6.1.28",
-    "@leafygreen-ui/toggle": "^10.1.2",
-    "@leafygreen-ui/tokens": "^2.11.0",
-    "@leafygreen-ui/tooltip": "^12.0.0",
-    "@leafygreen-ui/typography": "^19.3.0",
+    "@leafygreen-ui/badge": "^9.0.2",
+    "@leafygreen-ui/banner": "^9.0.2",
+    "@leafygreen-ui/button": "^22.0.2",
+    "@leafygreen-ui/card": "^12.0.2",
+    "@leafygreen-ui/checkbox": "^14.0.2",
+    "@leafygreen-ui/code": "^16.0.2",
+    "@leafygreen-ui/combobox": "^11.0.2",
+    "@leafygreen-ui/confirmation-modal": "^6.0.2",
+    "@leafygreen-ui/emotion": "^4.0.9",
+    "@leafygreen-ui/guide-cue": "^7.0.2",
+    "@leafygreen-ui/hooks": "^8.3.4",
+    "@leafygreen-ui/icon": "^13.1.2",
+    "@leafygreen-ui/icon-button": "^16.0.2",
+    "@leafygreen-ui/info-sprinkle": "^4.0.2",
+    "@leafygreen-ui/leafygreen-provider": "^4.0.2",
+    "@leafygreen-ui/logo": "^10.0.2",
+    "@leafygreen-ui/marketing-modal": "^5.0.2",
+    "@leafygreen-ui/menu": "^28.0.2",
```

Comment:
<img width="1301" alt="Screenshot 2025-02-05 at 10 19 21 AM" src="https://github.com/user-attachments/assets/d67e013f-c0ba-4f54-ad2e-d82b8b80cf10" />

Noticed that the hotkey hover is now hard to see, maybe something we can fix? Not sure if there are any other occurrences outside of the focus mode.

Reply:
I agree this isn't ideal. This doesn't seem to have changed with this PR though as I'm seeing this with Compass v1.45.2 as well. I've created https://jira.mongodb.org/browse/COMPASS-8949 to track this separately.

Reply:
I'm not seeing this issue on v1.45.2 🤔 
<img width="591" alt="Screenshot 2025-02-11 at 10 44 35 AM" src="https://github.com/user-attachments/assets/bc2c8d93-1f83-4f58-8356-61cf5598bd6e" />

Reply:
Nice catch! Thanks